packages/from-to-location-picker/src/styled.js

```diff
@@ -14,6 +14,7 @@ export const FromToPickerSpan = styled.span`
 `;
 
 export const Button = styled.button`
+  background: none;
```

Comment:
Would inherit be better here? I'm not entirely sure the context of this change.

Reply:
Inherit what?

Reply:
Without the background override, the default behavior on Chromium browsers is to display these buttons gray. There is no other stylesheet that sets that anywhere else, so we have to do it here.

Reply:
Gotcha. Thanks for explaining.